dashboard/lib/services/script_seed.rb

```diff
@@ -588,6 +593,28 @@ def self.import_lessons_standards(lessons_standards_data, seed_context)
       LessonsStandard.joins(:lesson).where('stages.script_id' => seed_context.script.id)
     end
 
+    def self.import_lessons_opportunity_standards(lessons_opportunity_standards_data, seed_context)
+      lessons_opportunity_standards_to_import = lessons_opportunity_standards_data.map do |ls_data|
+        lesson_id = seed_context.lessons.select {|l| l.key == ls_data['seeding_key']['lesson.key']}.first&.id
+        raise 'No lesson found' if lesson_id.nil?
+
+        standard_id = seed_context.standards.select {|s| s.shortcode == ls_data['seeding_key']['opportunity_standard.shortcode']}.first&.id
+        raise 'No standard found' if standard_id.nil?
+
+        LessonsOpportunityStandard.new(
+          lesson_id: lesson_id,
+          standard_id: standard_id
+        )
+      end
+
+      # inefficiently delete all LessonsStandards using 1 query per lesson, and
+      # then re-import all LessonsStandards in a single query.
+      seed_context.lessons.each {|l| l.opportunity_standards = []}
```

Comment:
it should now be possible to improve on this by switching to using `destroy_outdated_objects` instead. in fact, by changing other join models to have and `id` column, it should be possible to improve seeding for those models too. This is tracked as https://codedotorg.atlassian.net/browse/PLAT-900